modules/aws-outposts-environment-info-aws.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-environment-info-aws_{context}"]
+= Obtaining information from your AWS account
+
+You can use the AWS CLI (`aws`) to obtain information from your AWS account.
+
+[TIP]
+====
+You might find it convenient to store some or all of these values as environment variables by using the `export` command.
+====
+
+.Prerequisites
+
+* You have an AWS Outposts site with the required hardware setup complete.
+
+* Your AWS Outpost is connected to your AWS account.
+//Not sure how to talk about this, check AWS docs.
+* You have access to your AWS account by using the AWS CLI (`aws`) as an administrator.
+
+.Procedure
+
+. List the Outposts that are connected to your AWS account by running the following command:
++
+[source,terminal]
+----
+$ aws outposts list-outposts
+----
+
+. Retain the following values from the output of the `aws outposts list-outposts` command:
+
+** The Outpost ID.
+
+** The Amazon Resource Name (ARN) for the Outpost.
+
+** The AWS Outpost availability zone.
++
+[NOTE]
+====
+The output of the `aws outposts list-outposts` command includes two values related to the availability zone: `AvailabilityZone` and `AvailabilityZoneId`. The value you use to configure a compute machine set that creates compute machines in AWS Outposts is `AvailabilityZone`.
+====
+
+. Using the value of the Outpost ID, show the instance types that are available in your AWS Outpost by running the following command. Retain the values of the available instance types.
++
+[source,terminal]
+----
+$ aws outposts get-outpost-instance-types \
+  --outpost-id <outpost_id_value>
```

Comment:
I had avoided including `aws outposts` output because it is external to OpenShift. If they change this command output some day, we might not know about it and have incorrect info here. 

Generally, I would not even want to include the `aws outposts` command itself for the same reason, but decided this risk of an outdated command was likely worth the tradeoff in user convenience of having the command presented here. I omitted the output to minimize the number of third-party things that can change.

All that said: if you think there's a super low chance that this AWS stuff ever changes, I'd be glad to add the sample output for additional user convenience :grinning:

Reply:
@jeana-redhat I agree with you. I believe it is fine to me to have only the command (w/o output).